src/constants/common/index.js

```diff
@@ -5,3 +5,7 @@ export const BASE_URL = 'https://backend-user-product-management.vercel.app';
 
 export const HEADER_AUTHORIZATION = 'Authorization';
 export const HEADER_GROUP_ID = 'x-request-id';
+
+export const INVALID_PHONE_NUMBER_ERROR = 'Phone Number must be 11-digit that starts with 09';
```

Comment:
Phone Number must be 11 digits and starts with 09.

Reply:
done.